.github/workflows/qodana-scan.yml

```diff
@@ -2,15 +2,25 @@ name: Qodana
 on:
   workflow_dispatch:
   pull_request:
+    paths:
```

Comment:
What does this change do?

Reply:
Only runs this qodana workflow when changes are made for java code